target/linux/ath79/generic/base-files/etc/hotplug.d/firmware/10-ath9k-eeprom

```diff
@@ -86,6 +86,10 @@ case "$FIRMWARE" in
 	;;
 "ath9k-eeprom-pci-0000:00:00.0.bin")
 	case $board in
+	adtran,bsap1920|\
+	adtran,bsap1925)
+		caldata_extract "art" 0x1000 0x440
+		;;	
```

Comment:
Can be consolidated with this case.

```
buffalo,wzr-hp-g450h|\
pcs,cap324)
```

Make sure to keep alphabetical sorting.